ONSdigital/takeon-ui
# Bdd date adjustment validation

# Description
Added tests for date adjustment values and refactored the existing the rsi tests to be able work after the period dates Q11 and Q12 are mandatory for validations to be triggered.

# How to test?
Export AWS environment variables
export dev url and cognito user pool name  
to run the tests use below command -
behave features/rsi/rsi_adjusted_response_validation.feature -D browser=headless to run date adjustment response tests only
or run all the rsi tests that will also run date adjustment response tests use this command -
behave features/rsi/ -D browser=headless
Note: Tests for blank date adjusted values will fail as the functionality defaulting blank date fields to expected date values is not developed will be able to use it as regression tests when the functionality is working for it and Value is Zero, PoPRM validation tests are failing raised bugs  for it SPP-1915 and SPP-1927.
If any of the steps aren't clear please let me know I can go through with you again!
